core/email.py

```diff
@@ -214,19 +190,16 @@ def email_to_admin(
         username=None,
         user_email=None,
         cc_user=True,
-        admin_user=None,
         request_tracker=False,
         html=False):
     """
     Send a basic email to the admins. Nothing more than subject and message
     are required.
     """
-    if admin_user:
-        sendto, sendto_email = admin_address(admin_user)
-    elif request_tracker:
-        sendto, sendto_email = request_tracker_address()
+    if request_tracker:
+        sendto = settings.ATMO_SUPPORT
     else:
-        sendto, sendto_email = admin_address()
+        sendto = settings.ADMINS
     # E-mail yourself if no users are provided
     if not username and not user_email:
         username, user_email = sendto, sendto_email
```

Comment:
I noticed that my changes break this.
How are we sending emails _from_ the user though? Should I just set `from_email=settings.EMAIL_HOST_USER`?

Reply:
Yeah that would be one way. There may be a way in django. You could probably find how they do it by searching for EMAIL_HOST_USER there

Reply:
What do you mean from the user? I don't think we do that.

Reply:
Okay, I'll dig deeper into that. So far I think we should scrap this whole part:
```
    # E-mail yourself if no users are provided
    if not username and not user_email:
        username, user_email = sendto, sendto_email
    elif not user_email:  # Username provided
        # TODO: Pass only strings, avoid passing 'User' object here.
        if isinstance(username, User):
            username = username.username
        user_email = lookupEmail(username)
        if not user_email:
            user_email = "%s@%s" % (username, settings.DEFAULT_EMAIL_DOMAIN)
    elif not username:  # user_email provided
        username = 'Unknown'
```
and just use `sender=settings.EMAIL_HOST_USER` because we cannot send emails from users' emails, and even if we could, I don't see a benefit. I will see what I think after I look into it more

Reply:
This `email_to_admin` function has `from_email=email_address_str(username, user_email)`
I looked some more and most uses of this function use `settings.ATMO_DAEMON` as the user argument, but `send_image_request_failed_email` function inputs atmo username and email

Reply:
I think I need to look into this a lot more before I can make any decisions and I don't want to be saying things that I don't know much about. I think I can refactor a lot more than I have so far